test/unit/objectLiteral.spec.ts

```diff
@@ -13,13 +13,34 @@ test.each([
     expect(lua).toBe(`local myvar = ${out}`);
 });
 
-test.each([{ input: "3", expected: 3 }])(
-    "Shorthand Property Assignment (%p)",
-    ({ input, expected }) => {
-        const result = util.transpileAndExecute(`const x = ${input}; const o = {x}; return o.x;`);
-        expect(result).toBe(expected);
-    },
-);
+describe("property shorthand", () => {
+    test("should support property shorthand", () => {
+        const result = util.transpileAndExecute(`
+            const x = 1;
+            const o = { x };
+            return o.x;
+        `);
+
+        expect(result).toBe(1);
+    });
+
+    // TODO:
```

Comment:
What is the TODO? Either solve it or add a clear description.

Reply:
This test shows currently invalid behavior, but probably it would be better to add it after it would be fixed